internal/sink/sink.go

```diff
@@ -222,46 +231,59 @@ func (whs WebhookSink) Send(secret, acceptType string, msg *wrp.Message) error {
 	return errs
 }
 
-func (whs WebhookSink) batchMsgs() {
+func (whs *WebhookSink) batchMsgs() {
 	msgs := []*wrp.Message{}
 	ticker := time.NewTicker(whs.BatchLinger) //TODO: setting to this for now - open to suggestions
-	select {
-	case msg := <-whs.ch:
-		msgs = append(msgs, msg)
-		if len(msgs) == whs.BatchMessages {
-			whs.sendBatch(msgs)
+	expired := false
+	for {
+		select {
+		case <-whs.readyCh:
```

Comment:
we should think about the readyCh as something to be used and thrown away (replaced), it's not something to store as state